### ISSUE TITLE
Create visualization of process timings

### ISSUE THREAD

**Issue by johanfol** [frmscoe/General-Issues#193]
In order to visualize the timings after a performance run:
* Login to Jupyter 
* Click: "new run has happened"
* Jupyter should then ingest all data in the "transactions" collection
* Take the _smallest_ transactions.report.timestamp property as "start time of run" and _largest_ transactions.report.timestamp as "end time" 
* Do some evaluations with the various different "prcgTm" properties (which is the time the different processors took, eg report.tadpResult.prcgTm = how long TADP took to do its processing) 
* Show Average, Mean, 90th-, 95th- and 99th percentile prcgTm's for each different processor
* Show E2E times (sum all prcgTm's for each processor per transactions) Average, Mean, 90th-, 95th- and 99th percentile prcgTm's for each transaction

Then we'd like Jupyter to save the above dataset / evaluations as a "performance run". The next time a performance run happens, we go back to step 1 above. So it is a manual process to say: a new run has happened. 

By saving the different runs (this ticket), this will enable us to write a separate report that can draw conclusions from different performance runs (another ticket). 

https://jupyter-docker-stacks.readthedocs.io/en/latest/#quick-start
https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html

**Comment by Justus-Ortlepp-frms** [comment]
Blocked by outstanding Jupyter implementation.

**Comment by johanfol** [comment]
related to https://github.com/frmscoe/General-Issues/issues/187
